token/core/common/authrorization.go

```diff
@@ -72,14 +75,14 @@ func NewAuthorizationMultiplexer(ownerships ...Authorization) *AuthorizationMult
 }
 
 // IsMine returns true it there exists an authorization checker that returns true
-func (o *AuthorizationMultiplexer) IsMine(tok *token2.Token) ([]string, bool) {
+func (o *AuthorizationMultiplexer) IsMine(tok *token2.Token) (string, []string, bool) {
```

Comment:
We could have an alias `WalletID` to make it clear what it returns

Reply:
okay